src/lib/features/feature-toggle/feature-toggle-strategies-store.ts

```diff
@@ -514,6 +514,130 @@ class FeatureStrategiesStore implements IFeatureStrategiesStore {
         };
     }
 
+    // WIP copy of getFeatureOverview to get the search PoC working
+    async searchFeatures({
+        projectId,
+        userId,
+        queryString,
+    }: { projectId: string; userId?: number; queryString: string }): Promise<
+        IFeatureOverview[]
+    > {
+        let query = this.db('features');
+        if (projectId) {
```

Comment:
this is first divergence - we only query by project if project is provided, otherwise we do cross project search